src/main/java/gov/cabinetoffice/gap/adminbackend/security/WebSecurityConfig.java

```diff
@@ -43,6 +43,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                                 "/export-batch/{exportId:" + UUID_REGEX_STRING + "}/completed",
                                 "/grant-export-batch/{exportId:" + UUID_REGEX_STRING + "}/status",
                                 "/grant-export-batch/{exportId:" + UUID_REGEX_STRING + "}/s3-object-key",
+                                "/grant-export-batch/{exportId:" + UUID_REGEX_STRING + "}",
```

Comment:
we shouldn't need to add this here because I think we want to only allow authorised users to make requests to this endpoint